host/xtest/xtest_4000.c

```diff
@@ -2000,6 +2000,34 @@ static const uint8_t ciph_data_aes_cts_vect6_ctx[] = {
 	0x3b, 0xc1, 0x03, 0xe1, 0xa1, 0x94, 0xbb, 0xd8,
 };
 
+/* Test case for https://github.com/OP-TEE/optee_os/issues/1203 */
+
+#define ciph_data_aes_cts_issue1203_key ciph_data_aes_cts_vect1_key
+#define ciph_data_aes_cts_issue1203_iv ciph_data_aes_cts_vect1_iv
+
+static const uint8_t ciph_data_aes_cts_issue1203_ptx[80] = {
+	0x00
```

Comment:
The plaintext could be be a little more complicated to be able to detect if bytes are swapped or similar.